OneCommunityGlobal/HighestGoodNetworkApp
# XiaoW Improve loading speed of new tasks resources

# Description
Fixes bug list PROJECTS/TASKS/WBS COMPONENT 6 (priority high) :

Improve loading speed of New Tasks resources
- Other Links -> Projects -> WBS -> Pick any -> Add Task -> Choose Resource
- When adding “resources” to a new task, the resources list of people often doesn’t include everyone until I refresh.

## Main changes explained:
- Delete obsolete codes in AddTaskModal component for finding members, they are not being used since PR [#694](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/694#issue-1609123908)
- Update the way AddTaskModal component handles members, make it update in time
- optimize logic of handleFilter function in TagsSearch component

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. open incognito browser with no cached data
4. log as admin user
5. go to dashboard → Other Links → Projects → Pick your project → Pick your WBS → Click Add Task → type member name in Resource input
6. verify a list of members will show up soon as you type the first letter of member name without refreshing

## Note:
The members must be in the same project
